more/free-programming-cheatsheets.md

```diff
@@ -112,6 +112,7 @@
 * [Big O Notation Cheat Sheet](https://algodaily.com/lessons/big-o-notation-cheat-sheet)
 * [Data Structures and Algorithms Cheat Sheet - Cheatography](https://cheatography.com/burcuco/cheat-sheets/data-structures-and-algorithms/)
 * [Data structures and algorithms study cheatsheets for coding interviews](https://www.techinterviewhandbook.org/algorithms/study-cheatsheet/)
+* [Data structures and algorithms study cheatsheets for competitive programming](https://cp-algorithms.com)
```

Comment:
This is a book, not a cheat sheet. Please move it to DSA in subjects file

Reply:
Ok cool